<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgConfigTaxServer</class>
 <widget class="QDialog" name="DlgConfigTaxServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tax server</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QLineEdit" name="leAddress">
     <property name="inputMask">
      <string>###.###.###.###</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>IPv4 address</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lePort">
     <property name="inputMask">
      <string>####</string>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lePassword"/>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="chRemote">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Remote server</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
